data/mods/Xedra_Evolved/monsters/changeling.json

```diff
@@ -158,6 +159,221 @@
       "xe_cold_iron_stab_damage": 0
     }
   },
+  {
+    "id": "mon_xe_troll",
+    "type": "MONSTER",
+    "name": { "str": "troll" },
+    "description": "An enormous man-like form, wearing rough-spun clothing and almost twice as tall as a human.  It bears a club almost as big as a tree-trunk in one meaty, earth-brown hand.",
+    "default_faction": "changeling",
+    "copy-from": "mon_changeling",
```

Comment:
My concern with using copy-from here was first that probably it should have a higher bash defense than a regular changeling, but then I discovered something I worry would make even less sense.   Let me know if you need me to make Draugr giants and some kind of fungalized large changeling?  Or do zombify and fungalize not copy-from and I'm confused?

"zombify_into": "mon_draugr",
    "fungalize_into": "mon_cracked_mirror",
    "armor": {
      "bash": 6,
      "cut": 6,
      "stab": 6,
      "heat": 6,
      "bullet": 6,
      "electric": 2,
      "xe_cold_iron_cut_damage": 0,
      "xe_cold_iron_bash_damage": 0,
      "xe_cold_iron_stab_damage": 0
    }